spaces/models.py

```diff
@@ -108,6 +108,12 @@ class Unit(models.Model):
     hauki_resource_id = models.CharField(
         verbose_name=_("Hauki resource id"), max_length=255, blank=True, null=True
     )
+    rank = models.PositiveIntegerField(
+        blank=True,
+        null=True,
```

Comment:
Just wondering if it would make sense to make this non-nullable with default value of 0. It looks like `null` does not have any special meaning here and might just complicate front-end code if they have to deal with nulls.

From performance point-of-view it probably does not make a difference, unless if we use old version of PostgreSQL 🤪 

I leave it up to you ☺️

Reply:
I was thinking that if we order results with this `rank` then 1st 2st would be 1,2 so zeros 0 would be on top right? Those were my fri-yay thoughts :smile:   

But tuesday thoughts would be that  setting a value for those is needed if we end up using e.g `django-admin-sortable2` since it needs some values to those. But then they could be just the `pk`